website/www/site/content/en/contribute/release-guide.md

```diff
@@ -135,7 +135,20 @@ __NOTE__: When generating the key, please make sure you choose the key type as _
 
   Here, the key ID is the 8-digit hex string in the `pub` line: `845E6689`.
 
-  Now, add your Apache GPG key to the Beam’s `KEYS` file both in [`dev`](https://dist.apache.org/repos/dist/dev/beam/KEYS) and [`release`](https://dist.apache.org/repos/dist/release/beam/KEYS) repositories at `dist.apache.org`. Follow the instructions listed at the top of these files. (Note: Only PMC members have write access to the release repository. If you end up getting 403 errors ask on the mailing list for assistance.)
+* Add your Apache GPG key to the Beam’s `KEYS` file both in
```

Comment:
I'm not sure we need a "Run all commands manually" section at all if we have the script. Duplicating everything is cumbersome both to read and to edit. (BEAM-10048)

Reply:
Yeah, probably a good idea since the instructions just match the script. I'll remove the section and just leave a note to check the script for manual commands.

Reply:
Removed the manual steps, except for info on how to get a key ID for the GPG key, because the release candidate also requires that.